proposals/dao/fip_82.ts

```diff
@@ -0,0 +1,191 @@
+import { ethers } from 'hardhat';
+import { expect } from 'chai';
+import {
+  DeployUpgradeFunc,
+  NamedAddresses,
+  SetupUpgradeFunc,
+  TeardownUpgradeFunc,
+  ValidateUpgradeFunc
+} from '@custom-types/types';
+import { getImpersonatedSigner, validateArraysEqual } from '@test/helpers';
+import { tribeCouncilPodConfig, protocolPodConfig } from '@protocol/optimisticGovernance';
+
+// How this works:
+// Deployment
+// 1. DAO deploys an admin tier factory contract to deploy admin pods, specifically the Tribal Council
+// 2. DAO uses factory contract to create a Tribal Council pod, with empty members initially
+// 3. DAO deploys a protocol tier pod factory to deploy protocol pods
+// 4. DAO uses protocol tier pod factory to deploy the first protocol pod
+
+// Validation
+// 1. Validate admins
+// 2. Validate all pod members, proposers and executors
+// 3. Validate correct roles set on contracts
+
+// TODO: Remove once have SHIP tokens on Mainnet
+const mintOrcaToken = async (address: string) => {
```

Comment:
Haven't seen anything else referencing this Orca token, is this intentional? If so can you put a blurb in the description about it? Feels weird for us to be minting Orca tokens (?)

Reply:
Sure. It's a gating mechanism Orca uses for a slow beta rollout. You can only deploy pods successfully if you have some Orca tokens